lithium/lithium.py

```diff
@@ -141,7 +143,8 @@ def __init__(self):
         self.cutBefore = self.DEFAULT_CUT_BEFORE
 
     def readTestcaseLine(self, line):
-        cutter = b"[" + self.cutBefore + b"]?[^" + self.cutBefore + self.cutAfter + b"]*(?:[" + self.cutAfter + b"]|$|(?=[" + self.cutBefore + b"]))"
+        cutter = b"[" + self.cutBefore + b"]?[^" + self.cutBefore + self.cutAfter + b"]*(?:[" + self.cutAfter + \
+            b"]|$|(?=[" + self.cutBefore + b"]))"
```

Comment:
I agree :+1:

`+` is still required between string constants and variables though.